tests/qam-kgraft/update_kgraft.pm

```diff
@@ -153,7 +153,7 @@ sub post_fail_hook() {
     type_string("logout\n");
 
     # reconnect to svirt console
-    my $svirt = select_console('svirt');
+    my $svirt = select_console('svirt', await_console => 0);
```

Comment:
again, I think it doesn't make sense that you have to do this for every svirt, if you can just as well fix it in the function